site/docs/v1/tech/installation-guide/kickstart.adoc

```diff
@@ -430,10 +452,20 @@ And the usage in the kickstart file:
 
 === Reference
 
-The following environment variables  are built in:
+The following environment variables are built in:
 
 * `FUSIONAUTH_APPLICATION_ID` - The Id of the FusionAuth application
 * `FUSIONAUTH_TENANT_ID` - The Id of the default Tenant which is where the FusionAuth application resides
+* `FUSIONAUTH_LAMBDA[lambdatype]_ID` - The Id of the default reconcile Lambda for supported Identity Providers, available since version 1.20.1. This would be used to configure a corresponding Identity Provider. For example, for Twitter the variable name would be would be `FUSIONAUTH_LAMBDA[TwitterReconcile]_ID`. 
+
+The following reconcile Lambda Ids are available as `FUSIONAUTH_LAMBDA[lambdatype]_ID` values: 
+
+* `OpenIDReconcile`
+* `SAMLv2Reconcile`
+* `AppleReconcile`
+* `FacebookReconcile`
+* `GoogleReconcile`
+* `TwitterReconcile`
  
```

Comment:
We also added `FUSIONAUTH_FORM[adminUser]_ID` and `FUSIONAUTH_FORM[adminRegistration]_ID` to allow the user to configure forms. 

Normally you won't need these - because these are the defaults. However, if you were to create a new form, and then modify the default tenant or application in kickstart, and then create a new application or tenant and want to use these default forms, you'd need these variables.

Reply:
Awesome, I added these.